### PR TITLE
docs: this.model -> ctx.model

### PR DESCRIPTION
**Pull request by lzqmyb**
because the method of get model on context;

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ module.exports = mongoose => {
 
 // app/controller/user.js
 exports.index = function* () {
-  this.body = yield this.model.user.find({});
+  this.body = yield this.ctx.model.user.find({});
```
Comment (dead-horse) [inline README.md]: it's tow different controller style, read the [document](https://eggjs.org/zh-cn/basics/controller.html#如何编写-controller) to see details.

it is good to change to

```js
exports.index = function* (ctx) {
  ctx.body = yield tx.model.user.find({});
}
```

Comment (lzqmyb) [inline README.md]: you are right

**Comment by atian25** [comment]
you don't need to close, just edit -> commit -> push

**Comment by lzqmyb** [comment]
ok, I'll try

**Comment by atian25** [comment]
LGTM, thanks.

see https://eggjs.org/zh-cn/contributing.html for `PR title` & `commit message` style rule.

**Comment by lzqmyb** [comment]
thanks, too .
this is my first commit with open source project
feel good.
                  send by a chinese pritice write English ^_^

**Pull request merged**